agent/Makefile

```diff
@@ -30,9 +30,16 @@ release:                        ## Build static pmm-agent release binary (Linux
 	$(PMM_RELEASE_PATH)/pmm-agent --version
 	ldd $(PMM_RELEASE_PATH)/pmm-agent 2>&1 | grep -Fq 'not a dynamic executable'
 
+release-gssapi:                 ## Build dynamic pmm-agent release binary (for GSSAPI support)
+	CGO_ENABLED=1 go build -v -ldflags "$(VERSION_FLAGS)" -tags 'osusergo netgo static_build gssapi' -o $(PMM_RELEASE_PATH)/pmm-agent-gssapi
```

Comment:
Would it still make sense to have the `static_build` tag for a dynamic build? Is seems to work against the logic.